tar-conduit.cabal

```diff
@@ -30,20 +31,23 @@ library
   else
       other-modules: Data.Conduit.Tar.Unix
       build-depends: directory
+                   , safe-exceptions
```

Comment:
Probably better now to use `unliftio` and `UnliftIO.Exception`, but not a big deal.